tools/SetupFlow/DevHome.SetupFlow/Views/ConfigurationFileView.xaml

```diff
@@ -33,23 +40,183 @@
+            <DataTemplate x:Key="KeyValueTemplate">
+                <StackPanel Spacing="2"
+                            Orientation="Horizontal"
+                            Visibility="{Binding Value, Converter={StaticResource StringVisibilityConverter}}">
+                    <TextBlock Style="{ThemeResource BodyStrongTextBlockStyle}" Text="{Binding Key}" />
+                    <TextBlock Text=":" />
+                    <TextBlock IsTextSelectionEnabled="True" Text="{Binding Value}" TextWrapping="WrapWholeWords" />
+                </StackPanel>
+            </DataTemplate>
+            <Style TargetType="ListView" x:Key="KeyValueList">
+                <Setter Property="SelectionMode" Value="None" />
+                <Setter Property="Margin" Value="10,0" />
+                <Setter Property="ItemContainerStyle">
+                    <Setter.Value>
+                        <Style TargetType="ListViewItem" BasedOn="{StaticResource DefaultListViewItemStyle}">
+                            <Setter Property="Margin" Value="0" />
+                            <Setter Property="Padding" Value="0" />
+                            <Setter Property="MinHeight" Value="0" />
+                            <Setter Property="MinWidth" Value="0" />
+                        </Style>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+            <x:Double x:Key="SectionSpacing">10</x:Double>
+            <Thickness x:Key="BlockThickness">0,0,0,5</Thickness>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <i:Interaction.Behaviors>
+        <ic:EventTriggerBehavior EventName="Loaded">
+            <ic:InvokeCommandAction Command="{x:Bind ViewModel.LoadedCommand}" />
+        </ic:EventTriggerBehavior>
+    </i:Interaction.Behaviors>
+    
     <controls:SetupShell Orchestrator="{x:Bind ViewModel.Orchestrator}">
-        <Grid>
+        <Grid RowSpacing="{StaticResource SectionSpacing}">
             <Grid.RowDefinitions>
+                <RowDefinition Height="auto" />
                 <RowDefinition Height="auto" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
+
             <InfoBar
-                x:Uid="ms-resource:///DevHome.SetupFlow/Resources/ConfigurationFileWarning"
+                x:Uid="ConfigurationFileWarning"
                 IsOpen="True"
                 IsClosable="False"
                 Severity="Warning"/>
-            <ScrollViewer Grid.Row="1" HorizontalScrollBarVisibility="Auto">
-                <TextBlock
-                    FontSize="{ThemeResource CaptionTextBlockFontSize}"
-                    Padding="15"
-                    FontFamily="{StaticResource CascadiaMonoFontFamily}"
-                    Text="{x:Bind ViewModel.Content, Mode=OneWay}"/>
+
+            <ctControls:Segmented Grid.Row="1" SelectedIndex="0">
+                <ctControls:SegmentedItem x:Name="SummaryView" x:Uid="ConfigurationFile_SummaryView" />

```

Comment:
FYI SegmentedItem isn't recognized by narrator.  The narrator will not say
1. What position a button is in, including the size of set.
2. Narrator does not say "Selected"

Number 1 can be mitigated by telling the narrator what to say.  Look at AddRepoDialog.xaml.

Number 2 is an open issue with the maintainers.

Reply:
Good catch! Updated!